python/py-nbconvert/Portfile

```diff
@@ -91,6 +91,9 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-tinycss2 \
                     port:py${python.version}-traitlets
 
+    depends_run-append \
+                    port:pandoc
```

Comment:
Adding a runtime or library dependency requires increasing the revision, as documented in the MacPorts Guide.

Reply:
Thanks, done.